frontend/src/chatroom.js

```diff
@@ -163,7 +165,7 @@ class ChatRoom extends Component {
                 {this.props.user.discovered && (
                   <KnockedOutMessage message="You have been discovered" />
                 )}
-                {!this.props.user.eliminated && !this.props.user.discovered && (
+                {!this.props.user.eliminated && !this.props.user.discovered && !this.state.guessFormHide && (
                   <div>
```

Comment:
could have been shorter by just checking whether winner isn't null. But probably clearer to read this way